components/nft-sale-widget/layout/owned-nfts/owned-nfts.tsx

```diff
@@ -61,7 +61,7 @@ const OwnedNfts = (props: OwnedNftsProps) => {
   };
 
   if (props.isLoading || imageUrisLoading || copyBalancesLoading) {
-    return <SkeletonLayout isOwnedNft={!props.isOriginal} />;
+    return <SkeletonLayout isOwnedNft={true} />;
```

Comment:
I assume this hardcoding of true is because of how `<SkeletonLayout />` gets used?

Reply:
I am asking because the original code is based of whether it is showing 721s or 1155s. I assume that's what `isOriginal` means?

Reply:
yeah the previous code was only showing the skeleton for the copy nfts in the owned psyc but we want it to show up for both the original and copies hence the change